### DISCUSSION TITLE
Website css has become broken

### DISCUSSION THREAD

**Discussion by milorue** [tailwindlabs/tailwindcss#5184] [Help] [answered]
Hi,

I've been working on a side project for the last few months just off and on and recently without any changes that would've caused it my site went from looking like this:

![image](https://user-images.githubusercontent.com/3159469/128958114-bdd61dd2-bc4b-487f-a8f3-b4371e8899db.png)

to looking like this
![image](https://user-images.githubusercontent.com/3159469/128958213-9167c09b-53aa-426a-bd28-3426c817eba0.png)

They are different pages but because of the styling break I can't even navigate to the correct page to show the difference.

I've tried checking out older commits from my repo but all seem to persist the same styling issue. Any explanation or help is appreciated.

**Comment by simonswiss** [comment, marked as answer]
Hey!

If you want any help you'll need to share some code, it's impossible to troubleshoot from screenshots like this!

If I had to guess though, looks like your main section (which used to be below the navigation) is now in the same `flex` container as the navigation, which is why it's all stacking horizontally.

I bet you have a template somewhere that adds a wrapped with a flex class, or something like that.

Happy to help if you share some code in a public GitHub repo 👍

**Reply by milorue** [reply]
Hi sorry, I wrote this super late and was frustrated, I was able to fix it looks like either my version control or a rogue find and replace removed a bunch of `flex flex-col` from the codebase thanks anys
